src/server/actions/reward/calculateRewards.ts

```diff
@@ -126,6 +123,13 @@ export const calculateRewardsAction = async (contestId: string) => {
     throw new ServerError(`Rewards for this contest were already calculated.`)
   }
 
+  if (hasNoApprovedFindings) {
+    return {
+      rewards: [],
+      totalRewards: 0,
+    }
+  }
+
```

Comment:
If there are no valid findings found, we should just mark the contest as finished and return funds to the owner.